.github/workflows/special-tests.yml

```diff
@@ -199,6 +203,7 @@ jobs:
       include-success-outputs: ${{ inputs.include-success-outputs }}
       run-coverage: ${{ inputs.run-coverage }}
       debug-resources: ${{ inputs.debug-resources }}
+    if: ${{ inputs.default-branch == 'main' }}
```

Comment:
Are you sure this comparison makes sense? Default branch is `main` and you compare with main. This is always `true`?

Reply:
I assume the syntax must be like in https://github.com/actions/checkout/discussions/277

Reply:
Maybe the name could be misleading. We could likely change it to "target-branch" (PRs welcome :) ) .

This comes from https://github.com/apache/airflow/blob/main/dev/breeze/src/airflow_breeze/branch_defaults.py - and this value is different for `v2-10-test` targeted PRs than for `main` targeted PRs. This is how we distinguish whether the PR / canary build is in `main` or `2.10` or other line.

It's explained here: https://github.com/apache/airflow/blob/088242a911ba52f16a8c97df0d6dcb7e47ca1b73/dev/breeze/doc/ci/04_selective_checks.md#L4


![image](https://github.com/user-attachments/assets/ad498ce3-0e30-4646-b9ed-73932572a384)

Reply:
Thanks. Did not know that. Good to know. Then name really is a bit mis-leading.

Reply:
PRs are most welcome :)